include/seqan3/io/alignment_file/input.hpp

```diff
@@ -875,6 +878,8 @@ class alignment_file_input
      */
     //!\brief Buffer for a single record.
     record_type record_buffer;
+    //!\brief A larger (compared to stl default) stream buffer to use when reading from a file.
+    std::vector<char> stream_buffer{std::vector<char>(1'000'000)};
```

Comment:
Why not directly?
```suggestion
    std::vector<char> stream_buffer(1'000'000);
```

Reply:
@rrahn This syntax is not allowed by the compiler because it can be [ambigous](https://stackoverflow.com/questions/24836526/why-c11-in-class-initializer-cannot-use-parentheses) in some situations.

Reply:
right. that thing. Only this would work but does not make it better:
```cpp
std::vector<char> stream_buffer{1'000'000, std::allocator<char>{}};
```
So leave it as is.